tools/xcode/templates/readme.txt

```diff
@@ -82,12 +82,6 @@ FAQ
 
 
 
-  * I'm still using SFML 1.6. Can I use these templates anyway?
-
-    No, these templates were made for SFML 2 only.
```

Comment:
It's safe to assume no users are still using 1.6 these days. Let's just not bother mentioning it.